cockpit/src/subscriptions-view.jsx

```diff
@@ -177,128 +212,86 @@ class SubscriptionStatus extends React.Component {
+                                </DescriptionListGroup>
+                            }
+                        </DescriptionList>
+                    </CardBody>
+                </Card>
             );
         }
 
-        let text;
         let label;
         let action;
-        let insights;
-        let note;
-        let syspurpose;
-        let sla;
-        let usage;
-        let role;
-        let add_ons;
-        let syspurpose_status;
-        let isUnregistering = (this.props.status === "unregistering");
-        if (this.props.syspurpose["service_level_agreement"]) {
-            sla = (
-                <div>
-                    <label>
-                        { _("Service Level:  ") }
-                        <span className="value">{ _(String(this.props.syspurpose["service_level_agreement"])) }</span>
-                    </label>
-                </div>
-            );
-        }
-        if (this.props.syspurpose["usage"]) {
-            usage = (
-                <div>
-                    <label>
-                        { _("Usage:  ") }
-                        <span className="value">{ _(String(this.props.syspurpose["usage"])) }</span>
-                    </label>
-                </div>
-            );
-        }
-        if (this.props.syspurpose["role"]) {
-            role = (
-                <div>
-                    <label>
-                        { _("Role:  ") }
-                        <span className="value">{ _(String(this.props.syspurpose["role"])) }</span>
-                    </label>
-                </div>
-            );
-        }
-        if (this.props.syspurpose["addons"]) {
-            add_ons = (
-                <div>
-                    <label>
-                        { _("Add-ons:  ") }
-                        <span className="value">
-                            { _(String(subscriptionsClient.toArray(this.props.syspurpose["addons"]).join(", "))) }
-                        </span>
-                    </label>
-                </div>
-            );
-        }
-        if (this.props.syspurpose_status) {
-            syspurpose_status = (
-                <div>
-                    <label>
-                        { _("Status: ") }
-                        <span className="value">{ _(String(this.props.syspurpose_status)) }</span>
-                    </label>
-                </div>
-            );
-        }
-        syspurpose = (
-            <div>
-                <h2>{_("System Purpose")}</h2>
-                <div className="dl-horizontal">
-                    {syspurpose_status}
-                    {sla}
-                    {usage}
-                    {role}
-                    {add_ons}
-                </div>
-            </div>
-        );
+
         if (this.props.status === 'unknown') {
-            text = _("Status: This system is currently not registered.");
-            label = <label>{text}</label>;
+            label = _("System is currently not registered");
```

Comment:
It feels a bit cleaner as a message without having 'This' in there. But if you think it should say 'This system' I am happy to change it.

Reply:
IMHO either "This" or "The" is needed, as otherwise it feels something is lacking.

Reply:
Actually, I think that just 'Not registered' would be even better. It is now being used as value in a description list (see first picture, it would be next to `Subscription` in the first card). The same way it does not say 'This system is connected to Insights' just 'Connected to Insights'.
WDYT?

Reply:
I think that strings 'Not registered' and 'Connected to Insights' are OK, but I'm not UX designer nor native English speaker. :smiley:

Reply:
Changed to 'Not registered' as insights would show 'Not connected' and that feels correct :)